src/dash-playlist-loader.js

```diff
@@ -1,15 +1,64 @@
-import { EventTarget } from 'video.js';
+import { EventTarget, mergeOptions } from 'video.js';
 import mpdParser from 'mpd-parser';
 import {
+  refreshDelay,
```

Comment:
Might make sense to extract some of these common functions into their own module at some point (doesn't have to be in this PR).  I think it will make how things are composed a little bit clearer.